File: app/src/main/java/android/pdf4664/notepad/utilities/FileUtils.java
package android.pdf4664.notepad.utilities;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileInputStream;
import java.io.FileOutputStream;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class FileUtils {

    public static void copy(File file1, File file2) {
        String templateContent = fileContent(file1);
        writeToFile(file2,templateContent);
    }

    public static String fileContent(File file) {
        String ret = "";

        FileInputStream fileInputStream = null;
        try {
            fileInputStream = new FileInputStream(file);
            InputStreamReader inputStreamReader = new InputStreamReader(fileInputStream);
            BufferedReader reader = new BufferedReader(inputStreamReader);

            String line = "";
            while ((line=reader.readLine())!=null) {
                ret+=line;
                ret+="\n";
            }

            reader.close();
        } catch (Exception e) {
            e.printStackTrace();
        }

        return ret;
    }

    public static void writeToFile(File file, String content) {
        try {
            FileOutputStream outputStream = new FileOutputStream(file);
            OutputStreamWriter writer = new OutputStreamWriter(outputStream);

            writer.write(content);
            writer.flush();
            writer.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/org/pdf4664/notepad/utilities/FileUtils.kt
package org.pdf4664.notepad.utilities

import java.io.BufferedReader
import java.io.File
import java.io.FileInputStream
import java.io.FileOutputStream
import java.io.InputStreamReader
import java.io.OutputStreamWriter

object FileUtils {

    fun copy(file1: File, file2: File) {
        val templateContent = fileContent(file1)
        writeToFile(file2, templateContent)
    }

    private fun fileContent(file: File): String {
        var ret = ""

        var fileInputStream: FileInputStream? = null
        try {
            fileInputStream = FileInputStream(file)
            val inputStreamReader = InputStreamReader(fileInputStream)
            val reader = BufferedReader(inputStreamReader)

            var line = reader.readLine()
            while (line != null) {
                ret += line
                ret += "\n"
                line = reader.readLine()
            }

            reader.close()
        } catch (e: Exception) {
            e.printStackTrace()
        }

        return ret
    }

    private fun writeToFile(file: File, content: String) {
        try {
            val outputStream = FileOutputStream(file)
            val writer = OutputStreamWriter(outputStream)

            writer.write(content)
            writer.flush()
            writer.close()
        } catch (e: Exception) {
            e.printStackTrace()
        }

    }
}
